README.md

```diff
@@ -1,2 +1,15 @@
 # app-electron
 Desktop application for Zen Audio Player
+
+Listen to YouTube videos, without the distracting visuals.
+
+[![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/zen-audio-player/zen-audio-player.github.io?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
+
+[![Build Status](https://travis-ci.org/artforlife/app-electron.svg?branch=master)](https://travis-ci.org/artforlife/app-electron)
+
+
+
+## To Run the Desktop Version
+Clone this repository. Run `npm install` from within the cloned directory. Amongst other npm modules, this will install `electron-prebuilt` module. To launch the app, use `./node_modules/.bin/electron .` from within the directory. 
```

Comment:
Instead, we should add script to `package.json` so we can just do `npm start`. See what we did in the main repo